src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/XUnit/XUnitTestAssemblyFinishedCtorIntegration.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="XUnitTestAssemblyFinishedCtorIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.XUnit
+{
+    /// <summary>
+    /// Xunit.Sdk.TestAssemblyRunner`1.RunAsync calltarget instrumentation
+    /// </summary>
+    [InstrumentMethod(
+        AssemblyNames = new[] { "xunit.execution.dotnet", "xunit.execution.desktop" },
+        TypeName = "Xunit.Sdk.TestAssemblyFinished",
+        MethodName = ".ctor",
+        ReturnTypeName = ClrNames.Void,
+        ParameterTypeNames = new[] { "_", "_", "_", "_", "_", "_" },
```

Comment:
I didn't know we could do that 🎉

Reply:
Yeah, it's very useful in overloads if we don't use the parameter.